app/test/service.spec.ts

```diff
@@ -0,0 +1,109 @@
+import oasis from '@oasislabs/client';
+
+jest.setTimeout(20000);
+
+const GAS_LIMIT = '0xF42400';
+
+describe('Deployments', () => {
+  it ('should deploy non confidential', async () => {
+    let multisig = await oasis.workspace.Multisig.deploy(
+      ['b8b3666d8fea887d97ab54f571b8e5020c5c8b58'], 1,
+      {
+        header: {confidential: false},
+        gasLimit: GAS_LIMIT
+      }
```

Comment:
you can make this entire object a declared constant

Reply:
Getting a little nitpicky but ok.

Reply:
Actually I'd rather not since not all of the tests have the same headers. Some deploy with confidential, while others do not.